integration-tests/call-builder-return-value/lib.rs

```diff
@@ -113,6 +113,10 @@ mod call_builder {
     mod e2e_tests {
         use super::*;
         use incrementer::IncrementerRef;
+        use ink_e2e::{
+            ChainBackend,
```

Comment:
Why can't you import `E2EBackend` since it is a subtrait of both backend traits?

Reply:
you actually have to import the source trait that provides methods :|

Reply:
What's the point of having `E2EBackend` trait then?

Reply:
your e2e testcase will receive an object implementing this trait (instead of `subxt_client::Client` object as it is done now)